docker/Rocky.dockerfile

```diff
@@ -29,10 +29,8 @@ RUN dnf update -y && dnf install -y \
         hdf5-devel \
         libxml2-devel \
         boost-devel \
-        blas-devel \
```

Comment:
Were there only 2 packages to remove from the list for Rocky?

Reply:
Correct, only 2 to remove.  However, I did move the installation of `wget` to the `libxmlpp` stage because that is where it is needed - thanks for making me take a closer look.